tiptapy/templates/extras/creator_note.html

```diff
@@ -0,0 +1,15 @@
+<div class="creator-note-container">
+  <div class="creator-note-header">
```

Comment:
We have DG renderer for dg type post.
Similarly we can create comic renderer for comic type post.

In tiptapy we keep mostly renderers related to normal post, all renderers related to DG are in reactor
https://github.com/stckme/reactor/tree/master/app/templates/renderer/dg

i would purpose to keep renderers related to comic in reactor as well to maintain separation of concerns and consistency, but ofcourse open for discussion, in case we want to change gears and keep renderers at one place that is tiptapy. @ujjwalmishra09 @gauravr 

also we can consider overriding doc.html just for comic type and add attr related code there.

Ref: Renderer for DG type https://github.com/stckme/reactor/blob/master/app/renderers/digitalgoods.py

Reply:
We are rendering for episode and which is normal post, from rendering point of view there is only two type post, normal post and dg post

Reply:
From design it looks more like "dg" ptype rather than "blog" ptype.
and since we have custom blocks in "episode" ptype, i am suggesting custom renderer for "episode".